src/util/util.mjs

```diff
@@ -718,23 +735,17 @@ export const breakText = function(text, size, styles = {}, opt = {}) {
 
             if (lineHeight === undefined && textNode.data !== '') {
 
-                var heightValue;
-
                 // use the same defaults as in V.prototype.text
                 if (styles.lineHeight === 'auto') {
-                    heightValue = { value: 1.5, unit: 'em' };
+                    lineHeight = getLineHeight({ value: 1.5, unit: 'em' }, textElement.getBBox().height);
```

Comment:
The bounding box (`textElement.getBBox().height`) is calculated when the `lineHeight` is provided in pixels but the value is never used.

This should be optimized somehow e.g.
```
function getLineHeight(heightValue, textElement) {}
```